docs/Changelog.md

```diff
@@ -4,6 +4,112 @@ All notable changes to this project will be documented in this file.
+* Remove unused resources in TestAdapter by @Youssef1313 in [#4379](https://github.com/microsoft/testfx/pull/4379)
+* Improve some flaky tests by @Evangelink in [#4381](https://github.com/microsoft/testfx/pull/4381)
+* Fix flakiness on test Log_WhenAsyncFlush_StreamWriterIsCalledOnlyWhenLogLevelAllowsIt by @Evangelink in [#4394](https://github.com/microsoft/testfx/pull/4394)
+* Simplify adapter props and targets by @Evangelink in [#4397](https://github.com/microsoft/testfx/pull/4397)
+* Fix typo in analyzer message by @Youssef1313 in [#4452](https://github.com/microsoft/testfx/pull/4452)
+* Hide DataTestMethodAttribute from IntelliSense by @Youssef1313 in [#4465](https://github.com/microsoft/testfx/pull/4465)
+* Add API docs feedback template by @Youssef1313 in [#4687](https://github.com/microsoft/testfx/pull/4687)
+* Drop unused source gen related code by @Evangelink in [#4662](https://github.com/microsoft/testfx/pull/4662)
+* Don't make TestDataSourceDiscoveryOption obsolete by @Evangelink in [#4366](https://github.com/microsoft/testfx/pull/4366)
+* Drop dependency to System.ValueTuple by @Evangelink in [#4398](https://github.com/microsoft/testfx/pull/4398)
+* Remove AsyncTestProperty by @Youssef1313 in [#4725](https://github.com/microsoft/testfx/pull/4725)
+* Cleanup VB entrypoint by @Youssef1313 in [#4884](https://github.com/microsoft/testfx/pull/4884)
+* Fix doc about supported tfms by @Evangelink in [#4825](https://github.com/microsoft/testfx/pull/4825)
+* Fix MSTest metapackage README by @Youssef1313 in [#4887](https://github.com/microsoft/testfx/pull/4887)
+* Many cleanup PRs by @SimonCropp!
+
+### Artifacts
+
+* MSTest: [3.8.0](https://www.nuget.org/packages/MSTest/3.8.0)
+* MSTest.TestFramework: [3.8.0](https://www.nuget.org/packages/MSTest.TestFramework/3.8.0)
+* MSTest.TestAdapter: [3.8.0](https://www.nuget.org/packages/MSTest.TestAdapter/3.8.0)
+* MSTest.Analyzers: [3.8.0](https://www.nuget.org/packages/MSTest.Analyzers/3.8.0)
+* MSTest.Sdk: [3.8.0](https://www.nuget.org/packages/MSTest.Sdk/3.8.0)
+* Microsoft.Testing.Extensions.CrashDump: [1.6.0](https://www.nuget.org/packages/Microsoft.Testing.Extensions.CrashDump/1.6.0)
+* Microsoft.Testing.Extensions.HangDump: [1.6.0](https://www.nuget.org/packages/Microsoft.Testing.Extensions.HangDump/1.6.0)
+* Microsoft.Testing.Extensions.HotReload: [1.6.0](https://www.nuget.org/packages/Microsoft.Testing.Extensions.HotReload/1.6.0)
+* Microsoft.Testing.Extensions.Retry: [1.6.0](https://www.nuget.org/packages/Microsoft.Testing.Extensions.Retry/1.6.0)
+* Microsoft.Testing.Extensions.TrxReport: [1.6.0](https://www.nuget.org/packages/Microsoft.Testing.Extensions.TrxReport/1.6.0)
+* MSTest.SourceGeneration: [1.0.0-alpha.25112.8](https://www.nuget.org/packages/MSTest.SourceGeneration/1.0.0-alpha.25112.8)
+* MSTest.Engine: [1.0.0-alpha.25112.8](https://www.nuget.org/packages/MSTest.Engine/1.0.0-alpha.25112.8)
+
```

Comment:
Please add the new contributors section with:

* @aelij made their first contribution in https://github.com/microsoft/testfx/pull/4754
* @Sergio0694 made their first contribution in https://github.com/microsoft/testfx/pull/3848